### DISCUSSION TITLE
How to make NocoDB send email when date and time match today?

### DISCUSSION THREAD

**Discussion by WeT-Klb** [nocodb/nocodb#1122] [Discord Threads]
Is it possible to make NocoDB send an email when a special date and time is reached?
The scenario:
I would like to make a table with one field having the date and time and when that appointment is reached, i would like to get a notification. 
If possible, it even was better let´s say one day before - just like the reminder Outlook.
How to create this table and webhook? Something like `When field "Date" = Today then send email`.

**Comment by wingkwong** [comment]
Currently it's not possible.

**Reply by RelativeRon** [reply]
Is this the kind of thing where I could write a Workflow Automation?

**Reply by o1lab** [reply]
We will think about this going forward

**Comment by coutouly** [comment]
is it in progress ?
Is it possible to make NocoDB send an email when a special date and time is reached, the date is in a field of a table

**Comment by DoanTrungDuc1212** [comment]
Is it possible to automatically send emails when a date arrives in a noco field?
